examples/integrations/cdi/datasource-hikaricp/README.adoc

```diff
@@ -0,0 +1,108 @@
+= Hikari Connection Pool Integration Example
+
+== Overview
+
+This example shows a trivial Helidon MicroProfile application that
+uses the Hikari connection pool CDI integration, shows how to package
+it into a Docker image, and shows how to run it in a Docker container.
+It also shows how to run the Oracle database in a Docker container and
+connect to it using the application's container.
+
+== Notes
+
+To create a user-defined Docker network:
+
+```sh
+$ docker network create helidon-examples
+```
+
+To log in to the Oracle Container Registry (which you will need to do
+in order to pull Oracle database images from it):
+
+```sh
+$ docker login -u your.name@yourdomain.com -p yourpasswordhere container-registry.oracle.com
+```
+
+To run Oracle's `database/standard` Docker image in a Docker container
+named `oracle` connected to this network that publishes its ports to
+the host:
+
+```sh
+$ docker container run --name oracle --network helidon-examples --detach -it --publish 1521:1521 --publish 5500:5500 --shm-size=3g container-registry.oracle.com/database/standard:latest
```

Comment:
Why `docker container run` instead of `docker run` ?
If these are equivalent then this might be a source of confusion sincesay most users are used to `docker run`.

Reply:
Fixed; the `docker <noun> <verb>` form is Docker's preferred form.

Reply:
Seems like the `docker container` command is relatively new. I can't find the information that states that one form is preferred over the other, can you provide a link ?

I'd vote for documenting the old form since it's well known and can avoid potential confusion.

Reply:
See https://blog.docker.com/2017/01/whats-new-in-docker-1-13/#h.yuluxi90h1om; this is the new form so is the one that should be documented.

Reply:
Yet it's not mentioned in the docs. It's already 2 years old and it's the first time I see it.
I do think it's a potential source of confusion.

The instructions on container-registry.oracle.com does document "docker run".

Reply:
https://docs.docker.com/engine/reference/commandline/container_run/

Reply:
You mis-understood my comment. The documentation does not mention the preferred syntax.

Reply:
Clearly we need an arbiter; @barchetta what say you?